src/static/riot/competitions/editor/_competition_details.tag

```diff
@@ -320,6 +320,7 @@
                 }
             }
             self.update()
+            self.form_updated()
```

Comment:
we need this back ?

Reply:
Yeah, the dropdown has a default value. So it only calls `forum_updated()` when the competition doesn't have that default value. So if we have it there it will call `form_updated` 1 or 2 times, instead of 0 or 1 times. If it isn't called the form bugs out.

Reply:
ah should push this directly to develop then?